providers/git/src/airflow/providers/git/bundles/git.py

```diff
@@ -56,6 +57,7 @@ def __init__(
         subdir: str | None = None,
         git_conn_id: str | None = None,
         repo_url: str | None = None,
+        remove_git_repo_on_versions: bool = True,
```

Comment:
```suggestion
        remove_git_folder_on_versions: bool = True,
```

Maybe? Not sure this one matters.